controllers/cloud.redhat.com/utils/utils.go

```diff
@@ -59,15 +64,15 @@ func (u *Updater) Apply(ctx context.Context, cl client.Client, obj runtime.Objec
 	meta := obj.(metav1.Object)
 
 	if *u {
-		Log.Info("Updating resource", "namespace", meta.GetNamespace(), "name", meta.GetName(), "kind", kind)
+		//Log.Info("Updating resource", "namespace", meta.GetNamespace(), "name", meta.GetName(), "kind", kind)
```

Comment:
should this line be removed?

Reply:
I wasn't sure if we should or not - we log earlier - this is a very low level log